src/ServiceControl/Hosting/Commands/CustomLivetimes/HostBuilderLifetimeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Particular.ServiceControl.Commands
+{
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+
+    static class HostBuilderLifetimeExtensions
+    {
+        public static void SetupLifetime(this IHostBuilder hostBuilder, bool runAsWindowsService)
```

Comment:
Maybe either use the `UseXYZ` or the `AddXYZ` language instead of setup? For example

`UseCustomizedLifetime` or something similar or even `UsePersistenceInitializingLifetime`